plugins/src/webrtcsink/imp.rs

```diff
@@ -1749,29 +1744,28 @@ impl WebRTCSink {
         let mut state = element.imp().state.lock().unwrap();
 
         if let Some(consumer) = state.consumers.get_mut(peer_id) {
+            // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
+            // that it is not worth it below that threshold
             let fec_ratio = {
-                // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
-                // that it is not worth it below that threshold
-                if bitrate <= 2_000_000 || consumer.cc_info.max_bitrate <= 2_000_000 {
+                if bitrate <= DO_FEC_THRESHOLD || consumer.cc_info.max_bitrate <= DO_FEC_THRESHOLD {
```

Comment:
`consumer.cc_info.max_bitrate <= DO_FEC_THRESHOLD` should be unnecessary as bitrate can't be > max_bitrate?

Reply:
Makes sense. I've also noticed that `do_fec` wasn't considered here, so part of bitrate for fec was always substracted from the encoders bitrate (even if fec is disabled).